src/schedule/2021-01-26-some-antics-redwoodjs-tutorials.md

```diff
@@ -0,0 +1,19 @@
+---
+title: 'Some Antics: Reviewing RedwoodJS Tutorials with Anthony Campolo'
+type: Live Event
+date: 2021-01-26T12:00:00-06:00
+speakers:
+  - Ben Myers
+  - Anthony Campolo
+---
+
+<!-- Embed the Twitch stream -->
+<div id="twitch-embed" data-responsive-youtube--container></div>
+<script src="https://embed.twitch.tv/embed/v1.js"></script>
+<script type="text/javascript">
+	new Twitch.Embed('twitch-embed', {channel: 'someanticsdev'});
+</script>
```

Comment:
Nice! So cool that you can embed this right on the page!

I have a couple questions:

## Fullscreen?

When the event is live, do you know if we are able to fullscreen the window?
If not, we might need to apply some conveniences making this fit the window.

But also, maybe it's not a big deal because if someone's going to watch the whole thing, they know what to do.

## Video-on-demand?

Do you plan on recording these as a video-on-demand?
Or will it be live only?
If live-only, would it make sense to maybe switch the displayed content based on date?

Example:
* before and thru the event, show twitch embed
* after show Podcast subscribe/episode embed?

We could do that manually for now but just thinking thru how to keep this an actionable page for those who show up later

## Shortcode?

Is it worth making this a shortcode?  `{% twitch-channel 'someanticsdev' %}`

Reply:
- **Fullscreen?**

Yes, this embed is fullscreenable! It's even fullscreenable when the channel isn't streaming, which is pretty neat!

I'm not sure I even intend for people to use events.lunch.dev itself to view the stream — mostly, I mean this embed as a big callout that "this event isn't happening in the Discord like our other events!"

- **Video-on-demand?**

I have every intention of saving the Twitch recordings and uploading them to YouTube, and I'll happily swap the embed when the recording is up.

- **Shortcode?*

Love the idea of a shortcode, so long as we're planning on reusing it (which I guess begs the question of whether we anticipate more Twitch streams - Some Antics or otherwise - ending up on this calendar.

Reply:
sweet, sweet, sweet!

yeah, i think we'll have plenty of opportunities to use a `twitch` shortcode.

i have a channel ready for lunch.dev, when we feel like the time is right to share various events with a larger audience.